client/components/user-home.js

```diff
@@ -1,55 +1,98 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import {connect} from 'react-redux'
-import {PlayerAdd, PlayerList, GameCard, GameActionButton} from '../components'
+import {
+  PlayerAdd,
+  PlayerList,
+  EventCard,
+  EventActionButton
+} from '../components'
+import {fetchAllEvents} from '../store/event'
 
 /**
  * COMPONENT
  */
-export const UserHome = props => {
-  const {email} = props
+export class UserHome extends React.Component {
+  constructor(props) {
+    super(props)
+  }
 
-  return (
-    <div className="container">
-      <h5>Welcome, {email}</h5>
-      <div className="divider" />
-      <div className="section">
-        {' '}
-        <h6>Pending Games</h6>{' '}
-      </div>
-      <div className="divider" />
-      <div className="row">
-        {[1, 2, 3, 4, 5, 6, 7, 8].map(card => (
-          <GameCard key={card} type="pending" />
-        ))}
-      </div>
-      <div className="section">
-        {' '}
-        <h6>Completed Games</h6>{' '}
-      </div>
-      <div className="divider" />
-      <div className="row">
-        {[1, 2, 3, 4, 5, 6, 7, 8].map(card => (
-          <GameCard key={card} type="done" />
-        ))}
-      </div>
-      {/* <PlayerAdd />
+  componentDidMount() {
+    this.props.getUsersEvents()
+  }
+
+  render() {
+    const userEvents = Object.values(this.props.events.byId)
+    return userEvents ? (
+      <div className="container">
+        <h5>Welcome, {this.props.email}</h5>
+        <div className="divider" />
+        <div className="section">
+          {' '}
+          <h6>Pending Games</h6>{' '}
+        </div>
+        <div className="divider" />
+        <div className="row">
+          {/* Might Not Be the Best way to do this,
+        filter out the events of the appropriate type, then map over them with JSX components */}
```

Comment:
Exactly how I'd do it.  The other alternative is to write a selector in the reducer. 

So export the fuction getUserEvents(userID) which does all the filtering, that way it's just a simple MAP here (and we can reuse the filter logic!).